scan/src/pages/BlockHomePage.re

```diff
@@ -14,73 +14,83 @@ module Styles = {
       backgroundColor(Colors.grayHeader),
     ]);
 
+  let fullWidth = style([width(`percent(100.0)), display(`flex)]);
+
   let textContainer = style([paddingLeft(Spacing.lg), display(`flex)]);
 
   let proposerBox = style([maxWidth(`px(270)), display(`flex), flexDirection(`column)]);
 };
 
-let renderBody = (idx, (height, timestamp, proposer, totalTx, totalFee, blockReward)) => {
+let renderBody = (idx, block: BlockHook.Block.t) => {
+  let height = block.height;
+  let timestamp = block.timestamp;
+  let proposer = block.proposer->Address.toOperatorBech32;
+  let totalTx = block.numTxs;
+
   <TBody key={idx |> string_of_int}>
```

Comment:
I think we can just use block height as index @evilpeach ?

Reply:
Sure